134/feat/extension table/header/row (2)

elanalynn opened the pull request:
Closes #134 

- Create `ExtensionTable`, `ExtensionTableHeader`, and `ExtensionTableRow`
- Add custom `mount` helper to wrap tests in `i18nProvider`
- Add index files to `ActionSet` components and tidy up exports

https://user-images.githubusercontent.com/7654369/141871437-42c85ee7-b409-4c09-af52-7dd2a98f3193.mov

## 🎩

- `cd` into packages/ui-extensions-server-kit`
- Run `yarn` to install dependencies
- Run `yarn storybook`
- Storybook will open automatically on port 6006
- Review `ExtensionTable` components

alanthai commented:
Good changes. There's two visual inconsistencies with the original, though

![Screen Shot 2021-11-16 at 2 18 21 PM](https://user-images.githubusercontent.com/1744278/142051035-82aa98b1-0231-4424-b019-0583c07be5c0.png)

The rows are taller, though that might be because of the font size. Maybe we can set CSS `html {font-size: ...}` to align with what we have in the dev console app?

Also the hover styling seems to have a gap all around the row.

elanalynn commented:
@alanthai 
> The rows are taller, though that might be because of the font size. Maybe we can set CSS `html {font-size: ...}` to align with what we have in the dev console app?

✅  &nbsp; I matched the computed styles in the dev console app exactly. I adjusted the padding on row and header cells and adjusted the font size and weight. 

> Also the hover styling seems to have a gap all around the row.

✅ &nbsp; I also tweaked the hover effect to sit evenly between the top and bottom.
